contrib/drivers/lvm/lvm_test.go

```diff
@@ -356,4 +358,8 @@ var (
 		- currently set to     256
 		Block device           253:3
 	`
+	sampleVGS = `
```

Comment:
Please change the code style here.

Reply:
We should keep the actual commond out text format.